app/jobs/get_usps_proofing_results_job.rb

```diff
@@ -63,13 +33,17 @@ def perform(_now)
       enrollments_in_progress: 0,
       enrollments_passed: 0,
     }
+    @request_delay_in_seconds = IdentityConfig.store.
+      get_usps_proofing_results_job_request_delay_milliseconds / MILLISECONDS_PER_SECOND
+    @proofer = UspsInPersonProofing::Proofer.new
```

Comment:
I'd consider makign this an instance method and memoizing it, which might make it easier to stub if needed

```ruby
def proofer
  @proofer ||= UspsInPersonProofing::Proofer.new
end
``

Reply:
good call, done in 9e167ddcb909ffc74e98b8e0c424dbcd60530b99